dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A library that implements a layer on top of PKCS11 API"
+DESCRIPTION="The official header files for PKCS11 Standard were released \
+under a non-free license. libp11 does not include these header files, but \
+instead uses a rewritten pkcs11.h header file from the Scute project."
+HOMEPAGE="https://github.com/opensc/libp11"
+COPYRIGHT="1991, 1999 Free Software Foundation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/releases/download/libp11-$portVersion/libp11-$portVersion.tar.gz"
+CHECKSUM_SHA256="938bf73fc57ef8b543c6a2ceab617d7d83ccf60456241b73a61adb5ca37cd7c5"
+SOURCE_DIR="libp11-$portVersion"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libp11$secondaryArchSuffix = $portVersion
+	lib:libp11$secondaryArchSuffix = $portVersion
+	lib:libpkcs11$secondaryArchSuffix = $portVersion
+	lib:pkcs11$secondaryArchSuffix = $portVersion
```

Comment:
Please replace
```
	lib:libp11$secondaryArchSuffix = $portVersion
	lib:libpkcs11$secondaryArchSuffix = $portVersion
	lib:pkcs11$secondaryArchSuffix = $portVersion
```
by
```
	lib:libp11$secondaryArchSuffix = 2.4.3 compat >= 2
	lib:libpkcs11$secondaryArchSuffix
	lib:pkcs11$secondaryArchSuffix
```
BTW, **`libpkcs11.so`** is just a symlink to **`pkcs11.so`**, so maybe we should drop
```
	lib:libpkcs11$secondaryArchSuffix
```
I tried with and without that line, but "hp -S" did not complain with any of these variants.
Hmm, maybe it does not complain about libs which are not in $libDir. (That lib is in $libDir/engines/.)